### PR TITLE
[W6.4b][T15-B3]Darren Chin Jhun Pyng

### PR DESCRIPTION
**Pull request by DarrenCzen**
Updated DeveloperGuide.md with two Non-Functional Requirements

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -94,7 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
-
+5. Should be able to be used by any individual who has not used an address book before.
+6. Should not showcase any unnecessary and explicit content which is unrelated to the address book.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:98]: What does this mean?
Please try be very specific when writing NFRs

**Comment by okkhoy** [comment]
@DarrenCzen 
Some comments added. Please ack & close the PR after reading comments.

File: doc/DeveloperGuide.md (hunk 1 above)
Comment (DarrenCzen) [reply doc/DeveloperGuide.md:98]: Yes I will try to be more specific in the upcoming NFRs in my addressbook level4. Thank you for the recommendation!

**Comment by DarrenCzen** [comment]
Yes I will try to be more specific in the upcoming NFRs in my addressbook level4. Thank you for the recommendation!

**Pull request closed without merging**